providers/aws/fargate/cluster.go

```diff
@@ -223,6 +223,7 @@ func (c *Cluster) loadPodState() error {
 		pod.taskArn = *task.TaskArn
 		pod.taskStatus = *task.LastStatus
 		pod.taskRefreshTime = time.Now()
+		pod.taskRoleArn = taskDef.TaskRoleArn
```

Comment:
Please move this new line to between taskArn and taskStatus, to follow the same order they were declared in the struct. See my other comment.